kong/cmd/config.lua

```diff
@@ -80,7 +80,7 @@ local function execute(args)
   package.path = conf.lua_package_path .. ";" .. package.path
 
   _G.kong = kong_global.new()
-  kong_global.init_pdk(_G.kong, conf, nil) -- nil: latest PDK
+  kong_global.init_pdk(_G.kong, conf) -- latest PDK
```

Comment:
I would remove the comment as well, it is also related with plugin versioning.
```suggestion
  kong_global.init_pdk(_G.kong, conf) 
```